var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -438,17 +438,13 @@ def _test_examples(self):
             ("wave-eqn", [r"Max Error = 2", r"Evolved solution to time"]),
         ]
         for exe, expected in checks:
-            reason = "test: checking output of {0} for {1}".format(exe, expected)
-            self.run_test(
-                exe,
-                [],
-                expected,
-                installed=False,
-                purpose=reason,
-                skip_missing=True,
-                work_dir=self._extra_tests_path,
-            )
-
-    def test(self):
-        """Perform smoke tests."""
-        self._test_examples()
+            with test_part(
+                self,
+                "test_examples_" + exe,
+                purpose=f"test: checking output of {exe} for {expected}",
```

Comment:
`test: ` is already pre-pended to the test output so can be dropped here; otherwise, the log message for each test is `test: test: checking...`.